src/index.ts

```diff
@@ -1,13 +1,13 @@
 import {GraphQLClient} from 'graphql-request';
-import fetch from 'isomorphic-fetch';
+import {fetch as crossFetch} from 'cross-fetch';
```

Comment:
should `cross-fetch` be listed as a dependency?

Reply:
Yes. We still use cross-fetch in the case where a global fetch is not available.